changelogs/fragments/5341-newrelic-v2-api-changes.yml

```
@@ -0,0 +1,6 @@
+major_changes:
+  - newrelic_deployment - removed newrelic v1 API, added support for v2 API (https://github.com/ansible-collections/community.general/pull/5341).
```

The correct spelling seems to be "New Relic" (https://en.wikipedia.org/wiki/Newrelic)
```suggestion
  - newrelic_deployment - removed New Relic v1 API, added support for v2 API (https://github.com/ansible-collections/community.general/pull/5341).
```